docs/data/pages.ts

```diff
@@ -0,0 +1,31 @@
+import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
+
+const pages: MuiPage[] = [
+  {
+    pathname: '/toolpad',
```

Comment:
I think that we should change this. Shouldn't we expect on https://deploy-preview-542--mui-toolpad-docs.netlify.app/toolpad/ to see the marketing page?